test/eth.accounts.signTransaction.js

```diff
@@ -20,7 +33,8 @@ var tests = [
             to: '0xF0109fC8DF283027b6285cc889F5aA624EaC1F55',
             toIban: 'XE04S1IRT2PR8A8422TPBL9SR6U0HODDCUT', // will be switched to "to" in the test
             value: "1000000000",
-            data: ""
+            data: "",
+            common: common
```

Comment:
Adding `common` to all of these just keeps these tests as they are. If common wasn't here, the injections would be mis-aligned because there would be an extra call for network_id.

Reply:
We could add a property to the test cases to check if a ``common`` object is passed or not and inject the missing step for the ``net_version`` JSON-RPC call otherwise we would miss this branch in these unit tests.

Reply:
👍 

Yes! Added cases for missing common.